src/app/views.py

```diff
@@ -153,18 +153,27 @@ def activate(request, uid, token):
 
 
 class PersonViewSet(viewsets.ModelViewSet):
+    """
+        API endpoint that allows person to be viewed or edited.
```

Comment:
`API endpoint that allows` duplicate in each your ViewSet. Better write something like that
`Person ViewSet which allow list, create, retrieve, update and delete methods`